tests/utils.sh

```diff
@@ -10,6 +10,5 @@ test_compile() {
 
 run_for_input() {
     run timeout 5s "$(dirname "$1")/parser" < "$1"
-    [ "$status" -eq 0 ]
```

Comment:
I had to remove this check. It didn't occur to me previously that we might want to write tests that fail to parse. It should be enough to compare the output, since when the parsing fails, it'll probably never produce correct output.